spec/models/star_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+describe Star do
+
+  context "database reflects the model" do
+    it{should have_db_column(:user_id)}
+    it{should have_db_column(:snipet_id)}
+    it{should have_db_index(:user_id)}
+    it{should have_db_index(:snipet_id)}
+  end
+
+  context "validation" do
```

Comment:
also here, you should *describe* how `validation` works instead of using `context`